### ISSUE TITLE
ValidationError when extracting children of a block due unsupported sub-type equation

### ISSUE THREAD

**Issue by dylanjcastillo** [jheddings/notional#157]
_To determine to current version, run `python -m notional`_

Notional Version: 0.7.3

### Description ###

I'm getting a ValidationError when trying to extract the children of a block using `notion.blocks.children.list("XYZ")`.

### Steps To Reproduce ###

When I execute `notion.blocks.children.list("XYZ")`, I get the following  error message:

```
---------------------------------------------------------------------------
ValidationError                           Traceback (most recent call last)
Cell In[23], line 3
      1 page_id = "XYZ"
----> 3 result = extract_from_page(page_id)

Cell In[22], line 34, in extract_from_page(page_id)
     31 output = {"title": title } | get_metadata(page) | {"blocks": []}
     33 for block in notion.blocks.children.list(page_id):
---> 34     block_output = extract_from_block(block)
     35     if not block_output:
     36         continue

Cell In[22], line 14, in extract_from_block(block)
     12 if block.has_children:
     13     block_output["blocks"] = []
---> 14     for child in notion.blocks.children.list(block):
     15         processed_child = extract_from_block(child)
     16         if not processed_child: 

File ~/Documents/GitHub/test.venv/lib/python3.10/site-packages/notional/iterator.py:149, in EndpointIterator.__call__(self, **kwargs)
    145 self.page_num += 1
    147 page = self._endpoint(start_cursor=self.next_cursor, **kwargs)
--> 149 api_list = ObjectList.parse_obj(page)
    151 for obj in api_list.results:
...
File ~/Documents/GitHub/test/.venv/lib/python3.10/site-packages/pydantic/main.py:342, in pydantic.main.BaseModel.__init__()

ValidationError: 1 validation error for BlockList
results -> 16
  Unsupported sub-type: equation (type=type_error)
```

The block contains paragraphs, equations, tables, code, images, and headings. Interestingly, `notion.blocks.retrieve("XYZ")` works correctly, so the issue seems to be specific to the children generator.

### Expected Behavior ###

Being able to list the children on a given page.

**Comment by jheddings** [comment]
Thanks for the bug report.  It looks like it is finding the equation block at an unexpected time.  That type should be supported, so I'll look into this.

**Comment by jheddings** [comment]
Confirmed...  Notional is missing the `equation` block type.  It should be straightforward to add.  Hopefully before the end of the week.

Thanks for the report and sorry for the trouble!

**Comment by dylanjcastillo** [comment]
Oh great, thank you very much!

**Comment by dylanjcastillo** [comment]
Hi @jheddings,

Many thanks for fixing the issue. Any idea of when you'll release the new version of notional? Just wondering if it makes sense to wait, or to build it from source.

Thanks again!

Best,
Dylan

**Comment by jheddings** [comment]
> Many thanks for fixing the issue. Any idea of when you'll release the new version of notional? Just wondering if it makes sense to wait, or to build it from source.

I hope to get it out before the end of the week...  I'm just waiting for feedback on a new feature, which shouldn't take long.

**Comment by dylanjcastillo** [comment]
Thank you!

**Comment by jheddings** [comment]
This is now available in `notional-0.8.0`
